subsys/logging/Kconfig

```diff
@@ -424,6 +437,41 @@ config LOG_BACKEND_RTT_FORCE_PRINTK
 
 endif # LOG_BACKEND_RTT
 
+if LOG_BACKEND_SPINEL
```

Comment:
does that work? the if is above it's symbol definition.

Reply:
No it's not. I miss clicked and it worked for me probably thanks for cmake cache. Fixed